common/scripted_effects/wc_sacrifice_scripted_effects.txt

```diff
@@ -138,18 +138,59 @@ caltulate_PREV_offmap_old_gods_currency_insanity_effect = {
 				}
 				else_if = {
 					limit = { tier = KING }
-					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 300 } }
+					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 350 } }
 				}
 				else = {
 					limit = { tier = EMPEROR }
-					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 500 } }
+					PREV = { change_variable = { which = local_received_offmap_old_gods_currency_insanity_variable value = 650 } }
 				}
 			}
-			# Child
+			#Class
```

Comment:
Mention in the changelog that classes give points now.
Also, I think it's better to tie it to Combat Rating or something else. Like sacrificing a dragon aspect gives you less than a legendary druid.

Reply:
we can just give points for races that cant have classes

Reply:
I don't like this idea because every time we add a new race we need to address this effect while referring to Combat Rating doesn't need it.

Reply:
> Combat Rating doesn't need it.

And handles more cases: like you may have a shitty warrior trait, but you're Duelist, Brave, etc.

Reply:
what if youre sick?

Reply:
It makes more sense you give less. What if you're incapable? You're not a big threat to the Old Gods if you're sick or incapable, right?

Reply:
its about power not threat

Reply:
> its about power not threat

You have less power when you're sick.

Reply:
And if you have less power, you're a less threat.

Reply:
if you blind malfurion is he no longer powerful?

Reply:
I didn't say you're not. I said you're less. Blind Malfurion is less powerful than sighted Malfurion indeed.

Reply:
And Combat Rating represents it.

Reply:
or if ysera is incapable is sacrificing her to old gods no longer worth their favor?

Reply:
Again. I didn't say it. It's worth LESS favor because incapable Ysera is less powerful than not-incapable Ysera.

Reply:
i dont think it matters shes still an aspect

Reply:
Her combat experience also matters (where you get +1 Combat Rating per won duel), but the effect doesn't take it into account. And a lot of other stuff while Combat Rating reflects it all.

Reply:
with combat rating you can have a situation in which shes worth less than some random guy

Reply:
I wouldn't call him random if he reached dragon-tier. I don't think anything wrong here. If she's less powerful, she should give less.

Reply:
he didnt she just got injured

Reply:
plus the character you sacrifice might have gotten injured by you so that shouldnt matter to old gods

Reply:
> he didnt she just got injured

And? Why do you think the dragon-tier random guy should give less if he's more powerful than Yser... even injured? It's not fair!

Reply:
> plus the character you sacrifice might have gotten injured by you so that shouldnt matter to old gods

Yes, true, but in other cases, Combat Rating works fairer than the current situation.

Reply:
well cause shes just weakened temporarily

Reply:
plus combat rating in our mod is not really balanced

Reply:
> well cause shes just weakened temporarily

I may say the same thing about dragon age stages. Why whelps give less, it's temporary.

Reply:
> plus combat rating in our mod is not really balanced

So instead of balancing it, we create a situation where we need to balance 2 things: combat rating and these sacrificing values.

Reply:
Theres a difference between weakened state and potential power

Reply:
Also combat rating differs between classes

Reply:
> Theres a difference between weakened state and potential power

Define the difference. Whelp is weakened state I say.

> Also combat rating differs between classes

The warrior you mean? It gives +5.

Reply:
Also, incapable and maimed traits aren't temporary, you're going to reflect it in the calculations by traits?

Reply:
Cancer isn't temporary too. You may cure it, but the chances are very low.

Reply:
Genetic traits aren't temporary.

Reply:
Thats why the current way works best

Reply:
> Thats why the current way works best

Why? You don't give more points for strong characters and less for weak. It's not fair.

Reply:
Sacrificing Cenarius doesn't give more.

Reply:
It was just ranks before but that didnt make sense so it also checks class

Reply:
Cenarius already gets points for class

Reply:
The same amount as Malfurion though he's stronger?

Reply:
It's not resolved.

Reply:
Opened https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/1407.